invenio_previewer/templates/invenio_previewer/macros.html

```diff
@@ -54,7 +54,7 @@
         <td class="nowrap">{{ file.size }}</td>
         <td class="nowrap">
           <span class="pull-right">
-            <a class="btn btn-xs btn-default" href="{{ file.uri }}">
+            <a class="btn btn-xs btn-default" href="{{ url_for('invenio_records_ui.recid_download', pid_value=file.pid.pid_value, filename=file.file.key) }}">
```

Comment:
Can you make the file listing match this:

![screen shot 2016-08-31 at 08 40 53](https://cloud.githubusercontent.com/assets/1698163/18118697/cb3907dc-6f56-11e6-9106-2db631dbfcbb.png)

https://github.com/zenodo/zenodo/blob/master/zenodo/modules/records/templates/zenodo_records/box/files.html#L46-L65